### PR TITLE
[SNDVOL32] Add left and right balance icons

### PR DESCRIPTION
**Pull request by binarymaster**
## Purpose

Add left and right balance icons, which are already there.

JIRA issue: None

## Proposed changes

image

## TODO
- [x] Make icons show in ReactOS

### CODE REVIEW

File: base/applications/sndvol32/lang/bg-BG.rc [hunk 1]
```diff
@@ -53,6 +53,8 @@ BEGIN
     CONTROL "", -1, "static", SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE, 4, 19, 82, 1
     LTEXT "Ляво/Дясно", -1, 10, 25, 80, 8
     CONTROL "", IDC_LINE_SLIDER_HORZ, "msctls_trackbar32", TBS_HORZ | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 25, 35, 40, 17
+    CONTROL IDI_SPEAKER_RIGHT, -1, "static", SS_ICON | SS_REALSIZEIMAGE | WS_CHILD | WS_VISIBLE, 14, 38, 10, 10 
+    CONTROL IDI_SPEAKER_LEFT, -1, "static", SS_ICON | SS_REALSIZEIMAGE | WS_CHILD | WS_VISIBLE, 66, 38, 10, 10
```
Comment (HBelusca) [inline base/applications/sndvol32/lang/bg-BG.rc]: Syntax is: `CONTROL "", IDI_SPEAKER_LEFT, "static", ...`
See https://docs.microsoft.com/en-us/windows/desktop/menurc/control-control for more details.
Therefore I have no idea why it works in Win7+.

Comment (binarymaster) [reply base/applications/sndvol32/lang/bg-BG.rc]: Without `SS_REALSIZEIMAGE` on Windows:

image

On ReactOS still no icon displayed.

-----

Using `ICON IDI_ICON, -1, x, y [, width, height]` both on Windows and ReactOS:

image

Comment (learn-more) [reply base/applications/sndvol32/lang/bg-BG.rc]: This is because inside `AddDialogControl` the dialog resource is parsed, and it is parsing it wrong.

Comment (HBelusca) [reply base/applications/sndvol32/lang/bg-BG.rc]: I'm not sure it's clever to implement such function in the first place.

Comment (learn-more) [reply base/applications/sndvol32/lang/bg-BG.rc]: Yeah I don't see why it would need to do this,
makes it more complex and error-prone (as shown).

Comment (binarymaster) [reply base/applications/sndvol32/lang/bg-BG.rc]: image

Everything works now thanks to @learn-more's patch! :tada:

**Pull request merged**